packages/data-stores/src/launchpad-navigator/reducer.ts

```diff
@@ -0,0 +1,31 @@
+import { combineReducers } from '@wordpress/data';
+import type { LaunchpadNavigatorAction } from './actions';
+import type { Reducer } from 'redux';
+
+const activeChecklistSlug: Reducer< string | null | undefined, LaunchpadNavigatorAction > = (
+	state,
+	action
+) => {
+	switch ( action.type ) {
+		case 'LAUNCHPAD_NAVIGATOR_RECEIVE_ACTIVE_CHECKLIST_SLUG':
```

Comment:
It feels like these values for `action.type` should be in `constants.ts` to avoid duplicating the strings.